rero_ils/modules/documents/dojson/contrib/marc21tojson/model.py

```diff
@@ -2,6 +2,7 @@
 #
 # RERO ILS
 # Copyright (C) 2019 RERO
```

Comment:
I say you can change `Copyright (C) 2019 RERO` to `Copyright (C) 2021 RERO` as well.